nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -148,6 +157,7 @@ def __init__(self, model, config_list, optimizer=None):
                     types of nn.module you want to apply quantization, eg. 'Conv2d'
         """
         super().__init__(model, config_list, optimizer)
+        self.quant_grad = QATGrad
```

Comment:
where we use `self.quant_grad`?

Reply:
![image](https://user-images.githubusercontent.com/50486294/103497806-96a1d980-4e7d-11eb-900d-62d963711504.png)
This is used to clip backward gradient between wmin and wmax.